Add news section

imad10902 opened the pull request:
I have added a news section to the IHR website which fetches latest news on Internet Health from a News API. 

## Description

For now, I have used Free Reddit API but I also implemented a more precise News API which also contains visuals, but it is paid for production environment, so I just tested it and took the screenshots. If IHR decides to include this section and include a paid API, then I would implement it immediately because I have already done so. I have just kept the API Key open for now because it is freely available API but if i am supposed to use the paid one, I'll make use of environmental variables.  To add the news section, I made a new component called "NewsSection" which follows same pattern as of any other component in terms of layout. I just fetched the news data in JSON form and rendered cards which show the news. The only thing i did different than other components is that I used simple CSS to design this component which would not affect the functionality of the existing website in anyways. Also, for including the news section in the Navbar, I just shifted the buttons to a little right to maintain the Navbar user friendly after adding News section in the Navbar.

If the maintainers want any changes, I am ready to do the changes because I know this news section can be made very good under the guidance of maintainers.

## Motivation and Context

Having a news section about internet health on the IHR website can provide a valuable resource for users who are interested in staying up to date on current events and trends related to the internet health and security. 

## How Has This Been Tested?

I have not tested the component for now because it requires maintainers' review and there would be some necessary changes. I will unit test this "NewsSection" component using the Vue Test Utils (VTU). I unit tested other components using VTU, for those components I have created other PRs. To unit test this component, I would create a testing file, where I would import the component to be tested which is "NewsSection"  and import the required methods like "mount" from '@vue/test-utils'.
`import { mount } from '@vue/test-utils';`
Then I would use "mount" method to mount or render the "NewsSection" component for unit testing. This would return an instance which provides access to various methods to simulate user interactions with the component, retrieve and modify its props and data, etc.
`const wrapper = mount(NewsSection);`
Then I can use this wrapper to access all the items (card in this case) that are rendered using the "findAll" method and passing the class of the items (which is same as they are getting rendered using loop). 
`const items = wrapper.findAll('.card');`
Then I can use other methods to see if the news item (card) has required things using methods like "toContain()", "toBe()", etc. 

## Screenshots (if appropriate):
**Screenshot of the news section using paid News API:**
![Screenshot (414)](https://user-images.githubusercontent.com/105483337/228759778-eeb45825-b05b-45dd-bc4b-7878ac898347.png)

**Screenshot of the news section using Free Reddit API:**
![Screenshot (415)](https://user-images.githubusercontent.com/105483337/228760575-039ef06e-cd12-43f6-ad19-aaf8209b444e.png)


## Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [x] My code follows the code style of this project.
- [] My change requires a change to the documentation.
- [] I have updated the documentation accordingly.

thisiskaransgit commented:
Did you discuss this change with the maintainers?
If you have discussed it already, first open up an issue describing the new feature, why is it needed, what problem it solves, etc.
Please do not create a PR about adding a new change without asking the mantainers.
Also, the tests are failing.

imad10902 commented:
> 

Addressing your comment:
1) I have mentioned the motivation of making these changes above under "Motivation and Context" section.
2) Tests in news section are failing because it is not tested (I have mentioned it above) as it requires necessary changes and maintainers approval.
3) Yes, you are right I should have created issues first. I will take care of this next time.

Thank you.

romain-fontugne commented:
I don't think we need a news section or at least, as mentioned by @thisiskaransgit, we first need to discuss this in issues
